src/Act/CLI.hs

```diff
@@ -79,8 +84,10 @@ data Command w
 
   | HEVM            { spec       :: w ::: String               <?> "Path to spec"
                     , sol        :: w ::: Maybe String         <?> "Path to .sol"
+                    , vy         :: w ::: Maybe String         <?> "Path to .vy"
                     , code       :: w ::: Maybe ByteString     <?> "Runtime code"
                     , initcode   :: w ::: Maybe ByteString     <?> "Initial code"
+                    , sources    :: w ::: Maybe String         <?> "Path to sources .json"
```

Comment:
How hard would it be to use this json file to allow writing Act contracts that span multiple files?